src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -390,21 +390,35 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
 
 #Region "Buffer helpers"
 
+        Private Function NextAre(chars As String) As Boolean
+            Debug.Assert(Not String.IsNullOrEmpty(chars))
+            Dim n = chars.Length-1
```

Comment:
@pharring  I'll retest it with `NextAre(0, chars)` as this method's implementation.